fuse/ipns/ipns_test.go

```diff
@@ -216,8 +306,42 @@ func TestAppendFile(t *testing.T) {
 	}
 }
 
+func TestConcurrentWrites(t *testing.T) {
```

Comment:
maybe `TestConcurrentWritesToDifferentFiles` ? i expected it to write to same file